Testnet/NFTAuctionStore/NFTAuctionStore/NFTAuctionStore.cs

```diff
@@ -0,0 +1,228 @@
+﻿using Stratis.SmartContracts;
+using System;
+
+[Deploy]
+public class NFTAuctionStore : SmartContract //,INonFungibleTokenReceiver
```

Comment:
Think the interface should be uncommented or removed.

Reply:
It is not possible to uncomment it. ATM it is a kind of documentation.

Reply:
You can inline the interface with this contract to compile with a single file, or compile the entire project directory if it's in a different file. But either way, leaving as comment for documentation is fine too 👍